uwblueprint/project-read-backend
# add enrolment fields to FamilyDetailSerializer

### Notion ticket link



[Current enrolment section](https://www.notion.so/uwblueprintexecs/Current-enrolment-section-c4f6505ca371412eb09a6b3368294992) note: the edit functionality will be implemented later



### Implementation description

- added enrolment fields to the FamilyDetailSerializer
- added some options for randomly generating enrolment statuses



### Steps to test

1. create dummy data
2. Run [front end branch](https://github.com/uwblueprint/project-read-frontend/pull/42#issue-667599515) and make sure enrolment fields show up in the sidebar (there should be 3 enrolment fields)



### What should reviewers focus on?

- code quality

### Checklist
